json_operations.js

```diff
@@ -0,0 +1,372 @@
+
+var new_json = [
```

Comment:
Check the `const` data type, when we need to use it.
Also, check the `let` datatype. Now it is more common then `var`